### ISSUE TITLE
VGG models with BN are not fully trained.

### ISSUE THREAD

**Issue by KaiHoo** [pytorch/vision#2110]
## 🐛 Bug

The models in https://github.com/pytorch/vision/tree/master/references/classification are mostly trained with 90 epochs and the learning rate is reduced 0.1x every 30 epochs. Thus the 
learning rate is reduced only twice. This learning rate scheduler works for ResNets, but may not work for VGG models with BN  since the loss landscape may not be so good. Here are parts of my training logs:
```
Epoch 85: train loss 1.226, acc: 70.264%; val loss: 1.027, acc 74.322%; use 26.467 mins.
Epoch 86: train loss 1.225, acc: 70.296%; val loss: 1.019, acc 74.454%; use 26.469 mins.
Epoch 87: train loss 1.221, acc: 70.421%; val loss: 1.021, acc 74.498%; use 26.458 mins.
Epoch 88: train loss 1.218, acc: 70.437%; val loss: 1.024, acc 74.476%; use 26.538 mins.
Epoch 89: train loss 1.215, acc: 70.500%; val loss: 1.029, acc 74.386%; use 26.458 mins.
Epoch 90: train loss 1.186, acc: 71.155%; val loss: 1.008, acc 74.864%; use 26.343 mins.
Epoch 91: train loss 1.178, acc: 71.314%; val loss: 1.009, acc 74.916%; use 26.403 mins.
Epoch 92: train loss 1.175, acc: 71.388%; val loss: 1.007, acc 74.904%; use 26.474 mins.
Epoch 93: train loss 1.174, acc: 71.428%; val loss: 1.005, acc 74.960%; use 26.453 mins.
Epoch 94: train loss 1.169, acc: 71.557%; val loss: 1.006, acc 74.904%; use 26.409 mins.
Epoch 95: train loss 1.169, acc: 71.539%; val loss: 1.005, acc 74.988%; use 26.372 mins.
Epoch 96: train loss 1.167, acc: 71.562%; val loss: 1.006, acc 74.992%; use 26.315 mins.
Epoch 97: train loss 1.165, acc: 71.580%; val loss: 1.004, acc 74.998%; use 26.475 mins.
Epoch 98: train loss 1.164, acc: 71.626%; val loss: 1.004, acc 74.972%; use 26.416 mins.
Epoch 99: train loss 1.164, acc: 71.623%; val loss: 1.001, acc 75.046%; use 26.441 mins.
```

The validation accuracies in 85~89 epochs match the results shown in https://pytorch.org/docs/stable/torchvision/models.html. But can be 0.5% improved if trained with 10 more epochs (and one more learning rate decay). The 75% top 1 accuracy of VGG19-BN really surprised me.

I think it is better to train 100 epochs and reduce the learning rate every 30 epochs [1] or training 90 epochs and reduce the learning rate at 30, 60, 80 epochs [2].

[1] Group Normalization
[2] Training ImageNet in 1 Hour

**Comment by fmassa** [comment]
Hi,

Thanks for the information! Very valuable!

Those models have been trained a long time ago, and I'm unsure if we should change the pre-trained weights now, given that they are used as references for quite some time already.
